src/reducers/GlobalState.ts

```diff
@@ -0,0 +1,25 @@
+import { GlobalState } from '../store/Store';
+import { updateState } from '../utils/Reducer';
+import { GlobalActionKeys } from '../actions/GlobalActions';
+
+const INITIAL_GLOBAL_STATE: GlobalState = {
+  isLoading: false
+};
+
+// This Reducer allows changes to the 'serviceGraphFilterState' portion of Redux Store
```

Comment:
This comment looks like a copy-n-paste from another file and should be removed or re-worded, I think. This reducer doesn't change serviceGraphFilterState - it changes the global state. no?

Reply:
Yes, cut-n-paste error.